cli/src/formatting.rs

```diff
@@ -0,0 +1,128 @@
+use anyhow::{bail, Result};
+use serde_json::Value;
+
+use crate::types::{Agent, LinkExpression, PerspectiveExpression, SentPerspectiveMessage};
+
+pub fn print_prolog_results(results: Value) -> Result<()> {
+    match results {
+        Value::Bool(true) => println!("true ✅"),
+        Value::Bool(false) => println!("false ❌"),
+        Value::String(string) => println!("{}", string),
+        Value::Array(array) => {
+            println!("\x1b[90m{} results:", array.len());
```

Comment:
The log format seems confused to me, can you elaborate a bit?

Reply:
That's ANSI escape codes for color: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors